package.json

```diff
@@ -36,12 +36,15 @@
     "@babel/polyfill": "^7.4.4",
     "@babel/preset-env": "^7.2.3",
     "@babel/preset-react": "^7.0.0",
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
We might want to consider:

> We also recommend installing the jest-dom eslint plugin which provides auto-fixable lint rules that prevent false positive tests and improve test readability by ensuring you are using the right matchers in your tests

As a separate PR (or same, up to you!). I think it could be valuable enough if it actually spots false positive tests.

Reply:
One of the cool things about this library in particular is that incorporates ARIA attributes in its matchers. For example: `toBeRequired()` check `required` or `aria-required` attributes. Pretty cool.

Reply:
Created a new issue to track this effort: https://www.pivotaltracker.com/story/show/170393787